app/controllers/mandatarissen/persoon/mandataris.js

```diff
@@ -128,6 +128,22 @@ export default class MandatarissenPersoonMandatarisController extends Controller
     if (!this.model.mandataris.isActive) {
       return 'Het is niet mogelijk de status van een afgelopen mandaat aan te passen.';
     }
-    return '';
+
+    return 'Dit zal een nieuw mandaat starten met de gewijzigde situatie';
+  }
+
+  get toolTipTextCorrecting() {
+    if (this.isDisabledBecauseLegislatuur) {
+      return 'Tijdens het voorbereiden van een legislatuur is het niet mogelijk een mandaat in die legislatuur te bewerken.';
+    }
+
+    return 'Om fouten te corrigeren, bijvoorbeeld een typefout';
+  }
+
+  get warningTextOCMWLinkToGemeente() {
+    return `Let op! Bij het wijzigen van deze gegevens worden mogelijke koppelingen
```

Comment:
i'm creating a story to only show this warning when such a link exists. E.g. for old mandataris instances it wont exist